oak/server/oak_grpc_node.cc

```diff
@@ -64,8 +64,8 @@ void OakGrpcNode::CompletionQueueLoop() {
 
   Handle req_handle = FindChannel(kGrpcNodeRequestPortName);
   Handle rsp_handle = FindChannel(kGrpcNodeResponsePortName);
-  ChannelHalf* req_half = BorrowChannel(req_handle);
-  ChannelHalf* rsp_half = BorrowChannel(rsp_handle);
+  MessageChannelWriteHalf* req_half = BorrowWriteChannel(req_handle);
```

Comment:
Is there the possibility that the channels is destroyed while being borrowed (and not nullified)? Would it make sense to use a weak_ptr instead?